### PR TITLE
Initialized client app with create-react-app.

### LINKED ISSUE #10: Initialize front end react app
Use create-react-app to initialize the front end client app.

### PR DESCRIPTION
**Pull request by hoslack**
Closes #10

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (Karyum) [inline client/.gitignore]: if  you wanna keep the .env files like this it's fine by me but remember to name them they way they are named in here,
also move the .gitignore to the root project

File: client/src/logo.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (Karyum) [inline client/src/logo.svg]: youre not gonna need this fine just delete it

File: client/src/registerServiceWorker.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
```
Comment (Karyum) [inline client/src/registerServiceWorker.js]: youre also probably not gonna need this file might as well remove it

File: client/src/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import registerServiceWorker from './registerServiceWorker';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+registerServiceWorker();
```
Comment (Karyum) [inline client/src/index.js]: also since youre gonna delete the service worker file you also dont need line

Comment (Karyum) [reply client/src/index.js]: and the import at the top

**Review by Karyum — CHANGES_REQUESTED** [review]

**Comment by claireinez** [comment]
@hoslack Could you reword your original comment on this PR to "Closes #10" please? That way, GitHub will automatically close the relevant issue once it is merged into `master`.

**Review by Karyum — APPROVED** [review]
I love the github workflow so far go on you guys !! :heart:

**Pull request merged**